docs/self-managed/tasklist-deployment/data-retention.md

```diff
@@ -31,9 +31,9 @@ The default time between a process instance finishing and being moved to a dated
 
 In case of intensive Zeebe usage, the amount of data can grow significantly overtime. Therefore, you should consider the data cleanup strategy.
 
-Dated indices may be safely removed from Elasticsearch. "Safely" means only finished process instances are deleted together with all related data, and the rest of the data stays consistent. You can use Elasticsearch Curator or other tools/scripts to delete old data.
+Dated indices may be safely removed from Elasticsearch. "Safely" means only finished process instances are deleted together with all related data, and the rest of the data stays consistent.
 
-Users updating from Elasticsearch 7 to Elasticsearch 8 will encounter issues with the Elasticsearch Curator. To resolve this, Tasklist allows configuring an Index Lifecycle Management (ILM) Policy using the `archiver` configuration options:
+Users updating from Elasticsearch 7 to Elasticsearch 8 will encounter issues with the [Elasticsearch Curator](https://www.elastic.co/guide/en/elasticsearch/client/curator/current/index.html). To resolve this, Tasklist allows configuring an Index Lifecycle Management (ILM) Policy using the `archiver` configuration options, which is enabled by default:
```

Comment:
Can we link to this page we maintain in our docs as well? https://docs.camunda.io/docs/self-managed/operational-guides/update-guide/elasticsearch/7-to-8/

We may need to float some of this to the **Update guide** as well. I'm ok with this happening in a separate PR scope.